### PR TITLE
add information about user feedback notifications

### PR DESCRIPTION
**Pull request by dalnoki**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by jas-kas — APPROVED** [review]

File: docs/product/user-feedback/index.mdx [hunk 1]
```diff
@@ -75,4 +75,4 @@ If you don't have Sentry's default issue alert turned on, follow these steps to
 
 ![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)
 
-To be alerted when crash-report feedback comes in, set up Workflow Notifications as described [here](https://docs.sentry.io/product/alerts/notifications/#workflow-notifications).
+To be alerted when crash-report feedback comes in, set up Workflow Notifications as described [here](https://docs.sentry.io/product/alerts/notifications/#workflow-notifications). Also, please make sure that Crash Report Notifications are enabled in Settings > Projects > [Project Name] > User Feedback page.
```
Comment (michellewzhang) [inline docs/product/user-feedback/index.mdx:78]:
```suggestion
To be alerted when crash-report feedback comes in, set up Workflow Notifications as described [here](https://docs.sentry.io/product/alerts/notifications/#workflow-notifications). Also, please make sure that Enable Crash Report Notifications are enabled in Settings > Projects > [Project Name] > User Feedback.
```

Comment (vivianyentran) [reply docs/product/user-feedback/index.mdx:78]: Modified your suggestion a bit

Comment (jas-kas) [reply docs/product/user-feedback/index.mdx:78]: Lets remove the entire 'workflow notifications' sentence? We can simplify this 

Per https://sentry.slack.com/archives/C05J714QPRR/p1710967736899059?thread_ts=1710404908.615159&cid=C05J714QPRR

Comment (vivianyentran) [reply docs/product/user-feedback/index.mdx:78]:
```suggestion
To get notifications when crash-report feedback comes in, make sure to turn on "Enable Crash Report Notifications" in Settings > Projects > [Project Name] > User Feedback.
```
How's this?

Comment (jas-kas) [reply docs/product/user-feedback/index.mdx:78]: Looks good!

**Pull request merged**